app/packs/src/apps/mydb/elements/details/cellLines/analysesTab/AnalysesContainer.js

```diff
@@ -64,7 +71,13 @@ class AnalysesContainer extends Component {
       cellLineDetailsStore.cellLines(item.id).setChanged(true);
     }
     this.forceUpdate();
-  }
+  };
+
+  handleCommentTextChange = (e) => {
+    const { currentElement } = ElementStore.getState();
+    currentElement.container.description = e.target.value;
```

Comment:
- Is description exists always?
- Checking for valid value before saving and triming the value?